### PR TITLE
BZ-2022831 Clarified unique domain requirements for Ingress Controller

### PR DESCRIPTION
**Pull request by bscott-rh**
CP to 4.9+
Bugzilla link: https://bugzilla.redhat.com/show_bug.cgi?id=2022831
Doc preview: https://deploy-preview-41148--osdocs.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws-network-load-balancer.html#nw-aws-nlb-existing-cluster_configuring-ingress-cluster-traffic-aws-network-load-balancer

QE approval ✅

### CODE REVIEW

**Comment by bscott-rh** [comment]
@julienlim @racedo Here is a fresh PR with the version change. [Preview can be found here](https://deploy-preview-43493--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned)

**Comment by bscott-rh** [comment]
@lihongan Please take a look at this minor change to the Ingress Controller documentation. Thank you

**Comment by lihongan** [comment]
@bscott-rh it looks good. thank you!

File: modules/nw-aws-nlb-existing-cluster.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
           type: NLB
 ----
 <1> Replace `$my_ingress_controller` with a unique name for the Ingress Controller.
-<2> Replace `$my_unique_ingress_domain` with a domain name that is unique among all Ingress Controllers in the cluster.
+<2> Replace `$my_unique_ingress_domain` with a domain name that is unique among all Ingress Controllers in the cluster. This variable must be a sub-domain of the DNS name `<clustername>.<domain>`.
```
Comment (kowen-rh) [inline modules/nw-aws-nlb-existing-cluster.adoc:53]: "Subdomain" does not require hyphenation. :slightly_smiling_face:

**Review by kowen-rh — COMMENTED** [review]
One potential nitpick regarding the word "subdomain", but otherwise this looks good to me! :+1:

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: modules/nw-aws-nlb-existing-cluster.adoc (hunk 1 above)
Comment (bscott-rh) [reply modules/nw-aws-nlb-existing-cluster.adoc:53]: Thanks, fixed this.

**Pull request merged**